ibis/expr/tests/test_dereference.py

```diff
@@ -17,9 +17,30 @@ def dereference_expect(expected):
     return {k.op(): v.op() for k, v in expected.items()}
 
 
+def test_dereference_project():
+    p = t.projection([t.int_col, t.double_col])
```

Comment:
Let's avoid using this method. Can you `select` instead?

Reply:
Updated.